__tests__/app.test.js

```diff
@@ -183,6 +183,42 @@ describe("GET Tests", () => {
         });
     });
   });
+  describe("GET Users", () => {
+    test("should return a Status Code : 200", () => {
+      return supertest(app).get("/api/users").expect(200);
+    });
+    test("should return an array of users", () => {
+      return supertest(app)
+        .get("/api/users")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.users).toBeInstanceOf(Array);
+        });
+    });
+    test("should have all 4 users", () => {
+      return supertest(app)
+        .get("/api/users")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.users.length).toBe(4);
+        });
+    });
+    test("should return all users with the correct structure", () => {
+      return supertest(app)
+        .get("/api/users")
+        .expect(200)
+        .then((response) => {
+          response.body.users.forEach((users) => {
+            expect(users).toHaveProperty("username");
+            expect(users).toHaveProperty("name");
+            expect(users).toHaveProperty("avatar_url");
+          });
+        });
+    });
+    test("should return a Status Code : 404 when invalid", () => {
+      return supertest(app).get("/api/8_not_valid_name_8").expect(404);
```

Comment:
This is a standard bad path test that we may have already implemented, did this test auto pass? If we haven't previously tested for it we can keep it but put it in its own describe block as it's not related to the functionality of this endpoint 👍